### ISSUE TITLE
Signup: Long usernames are cut off in "Change Username" list

### ISSUE THREAD

**Issue by rachelmcr** [wordpress-mobile/WordPress-iOS#8961]
### Expected behavior

I expect on the Change Username to have a full preview of each username I could select.

### Actual behavior

With a very long username, the end of the username is cut off with an ellipsis and there's no way to view the full username. This is especially visible when the Display Text is set to a very large font.

### Steps to reproduce the behavior

1. Go to the Settings app > General > Accessibility > Larger Text and set a very large text size.
2. Sign up for a new account in the WordPress app.
3. On the epilogue screen, select the username to open the "Change Username" screen.
4. Search for a long username.
5. Note that long usernames are cut off; try rotating the screen or selecting usernames to view the full name. There's no way to view them.

##### Tested on iPhone 6, iOS 11.2.6, WPiOS 9.7.0.20180316

**Comment by ScoutHarris** [comment]
@nheagy , @folletto - any thoughts on this could/should be fixed? For reference - 

![username](https://user-images.githubusercontent.com/1816888/38060739-399f8b4e-32a9-11e8-9f54-f8b3ae7523ff.png)

**Comment by folletto** [comment]
The full username should already be visible in the top paragraph:

> Your current username is %@. With few exceptions, others will only ever see your display name, %@.

**Comment by rachelmcr** [comment]
That paragraph shows the current username, but the paragraph doesn't change as you select other usernames from the list. For example, in the screenshot above the paragraph would always say this:

> Your current username is scoutharrissites127.

The only way to update that text is to select a new username from the list, exit the Change Username screen, and then reopen it. That process completes the username change so there's no way to preview the username in that case (or undo it).

**Comment by folletto** [comment]
We can then make that text update, with a change of copy — that was also the original idea, I think we discussed otherwise at some point, that's why I wasn't 100% sure. :)

**Comment by rachelmcr** [comment]
Ah, I'd lost track of this one.

> Any advice for a new copy there?

As you mentioned, it could start out with the existing copy:

> Your current username is %@. With few exceptions, others will only ever see your display name, %@.

Then, when someone selects a different username from the list, it could update to something like this:

> You are about to change your username from %@ to %@. Tap "Back" to complete this change.

Since there isn't a cancel button, we should probably switch back to the original copy if they select their current username from the list.

**Comment by designsimply** [comment]
Still valid since the copy suggestion that shows at the top of the screen after selecting a suggested username still doesn't change:

> You are about to change your username from %@ to %@. Tap "Back" to complete this change.
> <sup>Source: https://github.com/wordpress-mobile/WordPress-iOS/issues/8961#issuecomment-480001814</sup>

<img src="https://user-images.githubusercontent.com/1119271/94722794-03548780-0315-11eb-9b79-b8ec858b0af8.png" width="270" alt="IMG_3851">
<p style="clear:both;"><sup>Tested with WPiOS 15.8.0.0 TestFlight beta on iPhone 6S iOS 13.7.</sup></p>

Testing notes: I tested and confirmed that long usernames are cut off in the "Change Username" screen if Larger Font settings are at the middle mark or below Settings.app > Accessibility > Display & Text Size > Larger Text. fwiw I noticed that anything higher than the middle option results in the text wrapping in the list of suggested usernames. ([1m20s](https://cloudup.com/cZtXvxt5vDc))

Long usernames are cut off|Unless font size is very large|Wrapping for very large fonts
---|---|---
![IMG_3843](https://user-images.githubusercontent.com/1119271/94722348-6e518e80-0314-11eb-9916-bac9094107a5.png)|![IMG_3840](https://user-images.githubusercontent.com/1119271/94722443-8b865d00-0314-11eb-85ec-dab9b5d499bb.png)|![IMG_3848](https://user-images.githubusercontent.com/1119271/94722495-9e009680-0314-11eb-970f-03385f33c8bd.png)

<p style="clear:both;"><sup>Tested with WPiOS 15.8.0.0 TestFlight beta on iPhone 6S iOS 13.7.</sup></p>
